pkgs/by-name/sw/sway-easyfocus/package.nix

```diff
@@ -15,16 +15,17 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "sway-easyfocus";
-  version = "0.2.0";
+  version = "0.2.0-unstable-2025-01-15";
```

Comment:
Why move to unstable?  If there's a feature you need we can get upstream to make another release.  Also see my comment about window swapping being part of 0.2.0.

Reply:
I agree with you, didn't see your pull request. I'm closing this